client/dist/error/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
What does this do? I've never seen it before.

Reply:
oh I created a page to handle invalid listings ^^